tfhe/src/core_crypto/entities/packed_integers.rs

```diff
@@ -4,7 +4,7 @@ use crate::conformance::ParameterSetConformant;
 use crate::core_crypto::backward_compatibility::entities::packed_integers::PackedIntegersVersions;
 use crate::core_crypto::prelude::*;
 
-#[derive(Clone, serde::Serialize, serde::Deserialize, Versionize)]
+#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize, Versionize)]
```

Comment:
Why do you need to derive Debug, Eq, PartialEq?

Reply:
Maybe we don't need Debug. I added Eq and PartialEq so we can compare `CompressedCiphertextList` and make sure GPU and CPU's compression are identical. I just pushed a modification to `to_cuda_compressed_ciphertext_list` doctest doing that.

Reply:
we can't guarantee that if we have PBSes applied right ? as we may have some slightly different noise depending on fft impls ?

Reply:
Current implementation only has a PBS for decompression. Compression's result should be exactly the same.